wiki/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3.2"
```

Comment:
Would it be possible to bump to 3.8? And possibly move the file to the root directory (and rename it `docker-compose-wiki.yml`)?

Reply:
Of course. I would propose to rename it to `docker-compose-doc.yml` to be consistent with the directory name (`doc` as per your request).